panels/config.py

```diff
@@ -11,7 +11,13 @@
 c.EVENT_BOOKED = {'colspan': 0}
 c.EVENT_OPEN   = {'colspan': 1}
 
-c.PANEL_ROOMS = [getattr(c, room.upper()) for room in c.PANEL_ROOMS]
+invalid_panel_rooms = [room for room in c.PANEL_ROOMS if not getattr(c, room.upper(), None)]
+
+for room in invalid_panel_rooms:
+    log.warning('panels plugin: panels_room config problem: '
+                'Ignoring "' + room.upper() + '" because it was not also found in [[event_location]] section.')
+
+c.PANEL_ROOMS = [getattr(c, room.upper()) for room in c.PANEL_ROOMS if room not in invalid_panel_rooms]
```

Comment:
I think this is fine as-is.  Though for log messages you can say

```
log.warning('I want {!r} to be quoted', room.upper())
```

and then you don't have to muck around with ``"' + room.upper() + '"``